dictionaries/CallMap_historical.php

```diff
@@ -16383,6 +16383,7 @@
     'xdebug_get_profiler_filename' => ['string|false'],
     'xdebug_get_stack_depth' => ['int'],
     'xdebug_get_tracefile_name' => ['string'],
+    'xdebug_info' => ['mixed', 'category='=>'string'],
```

Comment:
`xdebug_info` was introduced in Xdebug 3. Xdebug 3 is only compatible with PHP 7.2 (https://xdebug.org/docs/compat#versions), and `CallMap_historical` documents the available functions for PHP 7.0. So I'd suggest moving `xdebug_info` from here to `CallMap_72_delta`.

Reply:
changed that!